app/controllers/application_controller/filter.rb

```diff
@@ -450,14 +450,10 @@ def adv_search_button
       if ["delete", "saveit"].include?(params[:button])
         if @edit[:in_explorer] || x_active_tree == :storage_tree
           tree_name = x_active_tree.to_s
-          if "configuration_manager_cs_filter_tree_tree" == tree_name
-            page.replace_html("#{tree_name}_div", :partial => "provider_foreman/#{tree_name}")
-          else
```

Comment:
Yeah, this is worth removal.

But it was added recently (d63b47ee30ed200fc2311bcb83bcca0c39afb009).